phpBB/includes/functions_user.php

```diff
@@ -26,8 +26,9 @@
 * @param array &$user_id_ary The user ids to check or empty if usernames used
 * @param array &$username_ary The usernames to check or empty if user ids used
 * @param mixed $user_type Array of user types to check, false if not restricting by user type
+* @param bool $update_references If false, the supplied array is unset and appears unchanged from where it was called
 */
```

Comment:
Can you also add a `@return null` to the doc block while you're at it?

Reply:
Not a problem, will do.

Reply:
I noticed this yesterday, scroll down a bit further to 
https://github.com/phpbb/phpbb/pull/5457#issuecomment-451495389 😃

Reply:
Oh my bad 😵